rust-to-c/build.rs

```diff
@@ -1,5 +1,7 @@
 extern crate gcc;
 
 fn main() {
-    gcc::Config::new().file("src/double.c").compile("libdouble.a");
+    gcc::Build::new()
+                .file("src/double.c")
+                .compile("libdouble.a");
```

Comment:
By the way, it is said in [the docs](https://docs.rs/gcc/0.3.54/gcc/struct.Build.html#method.compile) that
> The Rust compilier will create the assembly with the lib prefix and .a extension

Does it mean we can remove it from here?

Reply:
Indeed we can! The `gcc` crate has actually also been updated to `cc` 1.0, so want to update that here as well/

Reply:
@alexcrichton as we here maybe you could kindly help me with compiling shared library? :)
I found `Build.shared_flag` method which adds `-shared` to `cc` step.
So, I use your example and the code is 
```
cc::Build::new()
        .file("src/double.c")
        .shared_flag(true)
        .compile("libdouble.so");
```
But `Build.compile` seems to add _lib_ prefix and _.a_ ending, that is why `ar` step has output file `liblibdouble.so.a`. Am I doing something wrong?
What would be the right way to compile shared library with `cc` crate?

Reply:
Ah so the `shared_flag` isn't actually intended on creating shared objects (that's not supported by `cc`), it was added... I forget why but I think it probably shouldn't exist...